### PR TITLE
[Misc] Transfer Nicholas Angle into the country somewhere

### PR DESCRIPTION
**Pull request by Lewiky**
> Met Sergeant: Well, you've always wanted a transfer to the ~~country~~ _E2E test folder_.
> Nicholas Angel: In twenty years or so, yes.
> Met Sergeant: Well done you.
> Nicholas Angel: (thinks briefly) Hang on, I don't remember telling you that.
> Met Sergeant: Yes, you did. (He leans over to his desk) You said (He cocks his head to one side and opens a corner of a thick document on his desk; reading from the document, quickly and quietly) "I'd love to settle in the ~~country~~ _E2E test folder_ sometime, Janine." (He closes his document and looks at Nicholas)

### CODE REVIEW

File: src/test/scala/temple/detail/PoliceSergeantNicholasAngel.scala [hunk 1]
```diff
@@ -18,9 +16,5 @@ import scala.io.StdIn
 object PoliceSergeantNicholasAngel extends QuestionAsker {
 
   /** Police Sergeant Nicholas Angel loves asking questions to the public in order to find out the answers he needs */
-  override def askQuestion(question: String): String = {
-    val answer = StdIn.readLine(Console.BLUE + question + "\n" + Console.RESET)
-    println(answer)
-    answer
-  }
+  override def askQuestion(question: String): String = "github.com/squat/and/dab"
```
Comment (jaylees14) [inline src/test/scala/temple/detail/PoliceSergeantNicholasAngel.scala:19]: Surely...
```suggestion
  override def askQuestion(question: String): String = "github.com/just/the/one/swan/actually"
```

**Review by jaylees14 — APPROVED** [review]
Beauty

**Review by willjvsmith — APPROVED** [review]
Nice

**Review by c-harding — APPROVED** [review]
Best PR name and description I’ve ever seen. Would be nice to see Jay’s suggestion applied

**Pull request merged**